openedx_webhooks/templates/github_community_pr_comment.md.j2

```diff
@@ -26,10 +26,6 @@ GitHub pull request interface. As a reminder,
     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)
     or indicated your institutional affiliation.
   {%- endif %}
-  {%- if not in_authors_file %}
```

Comment:
Let's get rid of `in_authors_file` completely.  We don't need to calculate it in the view any more, and then we don't need it in the next clause here in the template, and then we can simplify the template.

Reply:
We now have two consecutive clauses with "if not has_signed_agreement", so we can combine them.